datasafe-simple-adapter/datasafe-simple-adapter-spring/src/test/java/de/adorsys/datasafe/simple/adapter/spring/InectionForMinioTest.java

```diff
@@ -0,0 +1,50 @@
+package de.adorsys.datasafe.simple.adapter.spring;
+
+import de.adorsys.datasafe.business.impl.e2e.WithStorageProvider;
+import de.adorsys.datasafe.simple.adapter.api.SimpleDatasafeService;
+import de.adorsys.datasafe.simple.adapter.spring.factory.SpringSimpleDatasafeServiceFactory;
+import de.adorsys.datasafe.simple.adapter.spring.properties.SpringAmazonS3DFSCredentialsProperties;
+import de.adorsys.datasafe.simple.adapter.spring.properties.SpringDFSCredentialProperties;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.util.stream.Stream;
+
+
+// yes, this is correct. the factory will be instantieated via spring with the filesystem
+// but during runtime we create another factory with minio, right after we got the parameters from
+// the descriptor
+@ActiveProfiles("filesystem")
+
+@Slf4j
+public class InectionForMinioTest extends InjectionTest {
+
+    private static Stream<StorageDescriptor> minioonly() {
+        return allStorages().filter(storage -> storage.getName().equals(StorageDescriptorName.MINIO));
```

Comment:
not necessary just `return Stream.of(minio())`

Reply:
yep, contained in your patch anyway.